backend/routes/account/controller.py

```diff
@@ -22,7 +23,10 @@ def __init__(self, graph):
             version="v1",
         )
 
+    @auth_group("app-admins-group")
```

Comment:
Note that we'll probably want to move this decoration to the crud mapping so we don't have to rewrite each function here.